bin/faucet/src/server/challenge.rs

```diff
@@ -17,10 +17,17 @@ const CHALLENGE_ENCODED_SIZE: usize = 95;
 /// A challenge for proof-of-work validation.
 #[derive(Debug, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct Challenge {
-    pub(crate) difficulty: usize,
+    /// The target used to validate the challenge solution. A lower target makes the challenge more
```

Comment:
It might be worth to clarify that isn't the whole hash but rather the first 8 bytes